### ISSUE TITLE
Some questions about using different types of backbone

### ISSUE THREAD

**Issue by Neo233333** [fzh0917/STMTrack#15]
hello! Have you used different types of backbones for experiments（like ResNet or MobileNet）? And how effective is it？thank you for your reply !

**Comment by fzh0917** [comment]
We have tried to use ResNet-50 as the backbones, and the total performance on the GOT-10k benchmark is almost the same as the one using Inception-v3 (GoogLeNet), but the latter has lower FLOPs, so we use the latter as our backbones for all experiments. :)

**Comment by Neo233333** [comment]
Ok, thankyou!

**Comment by Neo233333** [comment]
hello！ I'm sorry to bother you again. Can you send me the pre-train ResNet50 model file or tell me where to get it? I want to do some research about the backbone. Thank you ! :)

**Comment by Neo233333** [comment]
Have you applied foreground-background label maps when you use Resnet50 as backbone? I don't find the code about  foreground-background label maps in resnet.py.  Sorry to bother you again and again :(

**Comment by fzh0917** [comment]
> hello！ I'm sorry to bother you again. Can you send me the pre-train ResNet50 model file or tell me where to get it? I want to do some research about the backbone. Thank you ! :)

The pretrained weights of ResNet-50 we used is downloaded from [here](https://github.com/pytorch/vision/blob/22ff44fd14139f2a056ad52b9bd109bd958089f3/torchvision/models/resnet.py#L28).

**Comment by fzh0917** [comment]
> Have you applied foreground-background label maps when you use Resnet50 as backbone? I don't find the code about foreground-background label maps in resnet.py. Sorry to bother you again and again :(

We did not merge the code of that experimental branch into the master/main branch, so you can't find it. I believe that you are able to add this functionality in the ResNet network according to the code logic of the Inception-v3 network.

**Comment by Neo233333** [comment]
OK！Thank you for your reply.
